README.md

```diff
@@ -1,262 +1,289 @@
-<h1 align="center">web-component-analyzer</h1>

```

Comment:
im guessing this entire-file-changed diff is because of line endings? do you know what we went from/to?

Reply:
This comes from pre-commit hook that did this change.
If you do a minor change in README and commit it, this should do the same. (I was able reproduce on a fresh clone, I hope it's not related to environment, Debian Linux on my case)
I think you should either commit line endings to README on master branch or update commit hook to avoid doing this change.